editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -1202,7 +1343,7 @@ void SpatialEditorViewport::_sinput(const Ref<InputEvent> &p_event) {
 						if (_edit.snap || spatial_editor->is_snap_enabled()) {
 
 							scale = Math::stepify(scale, spatial_editor->get_scale_snap());
-						}
+						}Transform t = se->original;
```

Comment:
Copy-paster error ?

Reply:
Still, if it is not useful you can remove the commented code.

Reply:
yup copy paste error :D totally forgot to remove this chunk sorry :)

Reply:
lol I think this copy paste error was before my PR hehe. Still huge commented block to removal :D